Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -219,6 +220,11 @@ private void OnEmptyTriggered(EntityUid uid, TriggerWhenEmptyComponent component
             Trigger(uid, args.EmptyGun);
         }
 
+        private void OnSpawnTriggered(EntityUid uid, TriggerOnSpawnComponent component, ComponentInit args)
+        {
+            Trigger(uid);
+        }
+
```

Comment:
This should be MapInit not ComponentInit because it will run in mapping mode and also whenever the game is reloaded.

Reply:
when is it useful to use a compinit?